laboratory-maniac.md

```diff
@@ -7,56 +7,59 @@ the cards that we are already playing.
 Laboratory Maniac serves as an alternative way to win within Doomsday 
 storm combo. It is a win condition that doesn't target the opponent 
-(Leyline of Sanctity), doesn't involve casting noncreature spells with 
-converted mana cost 4 or greater (Gaddock Teeg), and doesn't care about 
-your opponent's life total. Because Laboratory Maniac is a creature and 
-needs to be in play when you draw from an empty library, he is 
-vulnerable to removal spells like Swords to Plowshares, Lightning Bolt, 
-Pyroblast, and Abrupt Decay, which are all Legacy staples. The reason is 
-Laboratory Maniac serves as an additional strategy in Doomsday storm 
-combo, if you know your opponent is holding up Abrupt Decay, just build 
-a Tendrils pile that wins through that. Pyroblast is problematic for any 
-Ideas Unbound pile, Laboratory Maniac doesn't change that. Swords to 
+(Therefore playing around Leyline of Sanctity), doesn't involve casting 
+noncreature spells with converted mana cost 4 or greater (Gaddock Teeg),
+and doesn't care about your opponent's life total. Because Laboratory 
+Maniac is a creature, and needs to be in play when you draw from an empty 
+library, he is vulnerable to removal spells like Swords to Plowshares, 
+Lightning Bolt, Pyroblast, and Abrupt Decay which are all Legacy staples. 
+The reason we play Laboratory Maniac is he serves as an additional strategy in 
+Doomsday storm combo, if you know your opponent is holding up Abrupt Decay,
+just build a Tendrils pile that wins through that. Pyroblast is problematic 
+for any Ideas Unbound pile, Laboratory Maniac doesn't change that. Swords to 
 Plowshares and Lightning Bolt may be played through if you have 
-additional instant-speed draw, for example a 2nd Sensei's Divining Top 
+additional instant-speed draw, for example a second Conjurer's Bauble 
 in play or a Chromatic Sphere to win at mana-ability speed.
 
 Playing a 1-off Laboratory Maniac comes with a number of perks:
 - Win through any number of Leyline of Sanctity and/or Gaddock Teeg at very little extra cost.
 - Win through heavy discard with a number of extremely cheap pass the turn piles.
 - Win even if Burning Wish is extracted or named by Meddling Mage.
 - Win when you lack the resources to build sufficient storm for a Tendrils kill.
+- Sacrifices to Cabal Therapy Flashback in a pinch.
 
 And a couple of downsides:
 - Vulnerable to removal
 - Pretty dead card when you draw it pre-Doomsday.
 
-## Winning with Laboratory Maniac
+## Winning with Laboratory Maniac Using IU
 
 Now let's look into what winning with Laboratory Maniac looks like in 
-Doomsday storm combo. Assuming you have 1 Laboratory Maniac somewhere in 
+Doomsday. Assuming you have 1 Laboratory Maniac somewhere in 
 your maindeck, the standard piles are:
 ```
 GP in hand:
--> IU, SDT, LED, GP, LM BBB+1UU 4 life
+-> IU, CB, LED, GP, LM BBB+1UU (6) 4 life
 
-SDT in play:
--> IU, LED, GP, GP, LM BBB+1UU 4 life
+CB in play:
+-> IU, LED, GP, GP, LM BBB+1UU (6) 4 life
```

Comment:
This pile needs a CB instead of either GP, because you need to store a draw when using LED to pay for LM.